arviz/stats/stats.py

```diff
@@ -414,13 +466,23 @@ def loo(data, pointwise=False, reff=None):
                           the Observed RV in your model to make sure it returns element-wise logp.
                           """
             )
-        return pd.DataFrame(
-            [[loo_lppd, loo_lppd_se, p_loo, warn_mg, loo_lppd_i]],
-            columns=["loo", "loo_se", "p_loo", "warning", "loo_i"],
+        return pd.Series(
+            dict(
+                zip(
```

Comment:
I think using `index` keyword  here  is more straightforward than using dict and a zip. 

```
b = pd.Series(
            range(7),
            index =["loo", "loo_se", "p_loo", "warning", "loo_i", "kss", "loo_scale"]
        
)
```

Reply:
Yes, true, I did these in a hurry.